### PR TITLE
feat(test): seed script sniff test

### LINKED ISSUE #79: Add spec for seed file
I am seeing a lot of students breaking their app by making [`db/index`](https://github.com/FullstackAcademy/boilermaker/blob/b1c76f6023d50c7433030913acd7f9443e134b52/server/db/index.js#L6) export something other than `db` — e.g. an object containing `db` — and not updating the `db.sync` calls elsewhere. A good way of killing two birds with one stone would be to have a sanity test / sniff test for the seed file, which basically just confirms that the seed file runs without returning an exit code. Can be executed as a child process, with errors printed out.

### PR DESCRIPTION
**Pull request by glebec**
### Assignee Tasks

- [x] added unit tests (or none needed)
- [x] written relevant docs (or none needed)
- [x] referenced any relevant issues (or none exist)

Closes #79

### CODE REVIEW

**Comment by collin** [comment]
This seems worth the occasional "what is this" from students. Creates a learning opportunity for more inventive uses of testing.

Breaking seed files is a real issue during Sr. phase projects and this will prevent a broken seed from infecting master, which can create a full-team blocker.

**Comment by glebec** [comment]
Fixed indentation & resolved lockfile conflicts. (Sorry about weird merge, thought that a push had failed and did a local reset before I realized a commit had made its way upstream after all.)

**Comment by tmkelly28** [comment]
Couldn't we just `module.exports = seed`, require it into the test and assert that the promise it returns doesn't reject? Using cp here seems kind weird.

**Comment by glebec** [comment]
@tmkelly28 the seed file as it exists currently runs the `seed` function upon `require` / initialization. That script shuts down the database connection at the end, so you cannot both require the seed script _and_ invoke `seed` directly in tests. I think this may also have implications for running the seed module _alongside_ model tests, but I cannot recall if I actually ran into that as an issue or not.

We could wrap the existing call to `seed` in a `if (module === require.main)` check. That is a tiny bit more novel/complex for students, but probably how I would do this "for real."

I think at the time I was also just thinking that as a simple sanity check, I didn't want to tie the test to the seed file implementation at all – just know if it was even possibly to _run the module_ successfully. I suppose that directly `require`ing the module would provide the same information.

Ultimately I'm having trouble remembering why precisely I did it this way, I could be overlooking something that occurred to me at the time – or I could have just been using a hammer where a scalpel is more appropriate. I'll refactor this to use the `seed` function directly.

**Comment by glebec** [comment]
Lockfiles will be the death of me. Had to re-generate this one from scratch.

File: script/seed.js [hunk 1]
```diff
@@ -28,24 +30,27 @@ async function seed () {
...
-seed()
+// Execute the `seed` function, IF we ran this module directly (`node seed`).
+// `Async` functions always return a promise, so we can use `catch` to handle
+// any errors that might occur inside of `seed`.
+if (module === require.main) {
+  seed()
   .catch(err => {
-    console.error(err.message)
-    console.error(err.stack)
+    console.error(err)
```
Comment (glebec) [inline script/seed.js]: I didn't see any good reason to run separate `console.error` calls for `err.message` and `err.stack`. Default `console.error(err)` logs both the message and stack trace already.

**Pull request merged**